Makefile

```diff
@@ -72,7 +72,7 @@ updatedata: my.env
 	./docker/run_update_data.sh
 
 run: my.env
-	${DC} up webapp processor
+	${DC} up webapp processor webpack
```

Comment:
Is it the case that the webpack service produces output that's helpful to see while you're working on webapp stuff?

Reply:
Yes, it should produce error output when there's a JS error that breaks the webpack build, and output when a re-build runs to help confirm that the latest JS files are being served. Weirdly, that output isn't showing up with the current setup. I'm punting figuring out why that is to another time.